CMakeLists.txt

```diff
@@ -366,6 +366,8 @@ if(SOFA_BUILD_TUTORIALS)
     add_subdirectory(applications/tutorials)
 endif()
 
+#add_subdirectory(applications/tutorials)
+
```

Comment:
... same remark ;-)

Reply:
Yes, you're right, it slipped in in both cases!